SignalServiceKit/src/Messages/OWSMessageUtils.m

```diff
@@ -149,11 +151,46 @@ + (nullable TSQuotedMessage *)quotedMessageForMessage:(TSMessage *)message
         TSAttachment *_Nullable attachment =
             [TSAttachment fetchObjectWithUniqueID:attachmentId transaction:transaction];
         if (attachment) {
-            sourceFilename = attachment.sourceFilename;
-            contentType = attachment.contentType;
-            // Try to generate a thumbnail, if possible.
-            thumbnailData = [self thumbnailDataForAttachment:attachment];
-            hasAttachment = YES;
+            // If the attachment is "oversize text", try to treat it appropriately.
+            if (!hasText && [NSObject isNullableObject:attachment.contentType equalTo:OWSMimeTypeOversizeTextMessage] &&
```

Comment:
small readability nit: I like the existence of `[NSObject isNullableObject:equalTo:]` when it would otherwise be replaced by some 5 line nil check pattern, but it's kind of mouth full when it's not really necessary, so I'd rather not start using it all over the place.

In this case, even if contentType **were** nil, it'd do the right thing, but if you don't like the idea of calling the method on nil, you could flip it:

```
[OWSMimeTypeOversizeTextMessage isEqual:attachment.contentType]
```

Or we could avoid it altogether in Swift.

Reply:
Done.